package.json

```diff
@@ -21,6 +21,7 @@
   },
   "homepage": "https://github.com/IlonaMenkui/super-enigma-service#readme",
   "dependencies": {
+    "bunyan": "^1.8.12",
```

Comment:
I suggest you to use koa middleware for that. Please, look at https://github.com/koajs/bunyan-logger

Reply:
Also if you decide to use this middleware you will need to remove `bunyan` package from dependencies because `bunyan-logger` middleware already had it in deps.

Reply:
I use bunyan not only in koa layer, but also in my services, so maybe I need an explicit dependencie on it, what do you think?

Reply:
where? give me example please. for now I see it only in controllers

Reply:
It will be used in the feature.

Reply:
What should I do now in your opinion?

Reply:
Let's leave it as is for now